README.md

```diff
@@ -1,8 +1,22 @@
-# Query Loop Filter
+# Query Loop Filters
 
-Adds support for filtering the query loop block using post types and taxonomies.
+![Example of query loop filters](https://private-user-images.githubusercontent.com/494927/372880371-99de0b1b-a96c-4d6a-9bcc-e59c9fe18d4b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc4NzU5OTIsIm5iZiI6MTcyNzg3NTY5MiwicGF0aCI6Ii80OTQ5MjcvMzcyODgwMzcxLTk5ZGUwYjFiLWE5NmMtNGQ2YS05YmNjLWU1OWM5ZmUxOGQ0Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwMlQxMzI4MTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05OTY5ZmNhYjMwZjUwMzcxMTY5YTcwNTA0NzE3YjBhZmQxYTVjYTYzMTUyNTAxNzg1NTJjNGE0ZmU0NWM1YmE0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.cJO5wyEdtzqLCmtoV47Vg5bA3GIFK9LtJrT6wOgU5Lc)
```

Comment:
I don't think this is going to work for other users, there's a cleaner way of adding one that pulls from the repo, I don't remember the specifics though

Reply:
Cheers Tom. I had uploaded the image to an issue and then copied it in here... but I think you're right that this isn't ideal. I just tried copying the image straight into the Readme and looks like that works fine now :)